[[4096, "Error: \"\"/root/test-system/program_testing/source_solution/thread_2/solution.py\"\" is an invalid filename: rejected character: \"\"\"", null, 0.0014171600341796875]]